src/modules/users/components/UserTokenActivationButton/UserTokenActivationButton.tsx

```diff
@@ -24,22 +26,37 @@ const UserTokenActivationButton = ({ nativeToken, userLock }: Props) => {
   const totalBalance = inactiveBalance.add(activeBalance);
 
   return (
-    <>
-      <button type="button" className={styles.tokens}>
-        <span
-          className={`${styles.dot} ${
-            (inactiveBalance.gt(0) || totalBalance.isZero()) &&
-            styles.dotInactive
-          }`}
-        />
-        <Numeral
-          suffix={` ${nativeToken?.symbol} `}
-          unit={getTokenDecimalsWithFallback(nativeToken?.decimals)}
-          value={totalBalance}
-          truncate={3}
-        />
-      </button>
-    </>
+    <TokenActivationPopover
+      activeTokens={activeBalance}
+      inactiveTokens={inactiveBalance}
+      totalTokens={totalBalance}
+      lockedTokens={lockedBalance}
+      token={nativeToken}
+    >
+      {({ toggle, ref }) => (
+        <>
+          <button
+            type="button"
+            className={styles.tokens}
```

Comment:
I know this was not added here, but while you're at it...

Hovering the tokens should show a `cursor: pointer;`